### ISSUE TITLE
"homebrew_cask" in ES install triggers deprecation error failing install

### ISSUE THREAD

**Issue by sippsolutions** [valet-sh/valet-sh#71]
```
TASK [elasticsearch : ensure adoptopenjdk8 is installed
fatal: [local]: FAILED! => {
    "changed": false
}

MSG:

Updating Homebrew...
Error: Calling brew cask install is disabled! Use brew install [--cask] instead.
```

When I manually install the cask via `brew install --cask adoptopenjdk8` and remove the lines
```
- name: "ensure adoptopenjdk8 is installed"
  homebrew_cask:
    name: "homebrew/cask-versions/adoptopenjdk8"
    state: present
```
from `/usr/local/valet-sh/valet-sh/roles/elasticsearch/tasks/macos/install.yml` the install runs through.

**Comment by zelgerj** [comment]
https://github.com/ansible-collections/community.general/issues/1524

**Comment by zelgerj** [comment]
https://github.com/Homebrew/discussions/discussions/340#discussioncomment-232364

**Comment by dittertp** [comment]
We should use the general collection for ansible 2.9, because it seems unlikly that the fix is getting backported to ansible 2.9.

all we need to to is to add install command for the collections to the cli update process

ansible-galaxy colleciton install -r /usr/local/valet-sh/valet-sh/requirements.yml

**Comment by dittertp** [comment]
fixed in #81
